Bottom: 98bec02c4b6200b4005457840e95e074c280ca55
Top:    7ccafe8ab211fe0f57fe489d6d3d8f7a455d3a58
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 09:48:46 +0800

Refresh of autofs-5.0.7-fix-dead-LDAP-symbolic-link-when-LDAP-support-is-disabled.patch

---

diff --git a/modules/Makefile b/modules/Makefile
index 6e791cc..43c8a8d 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -76,7 +76,7 @@ install: all
 	-rm -f $(INSTALLROOT)$(autofslibdir)/mount_smbfs.so
 	ln -fs lookup_file.so $(INSTALLROOT)$(autofslibdir)/lookup_files.so
 	ln -fs lookup_yp.so $(INSTALLROOT)$(autofslibdir)/lookup_nis.so
-ifeq ($(SASL), 1)
+ifeq ($(LDAP), 1)
 	ln -fs lookup_ldap.so $(INSTALLROOT)$(autofslibdir)/lookup_ldaps.so
 endif
 	ln -fs mount_nfs.so $(INSTALLROOT)$(autofslibdir)/mount_nfs4.so
